department-of-veterans-affairs/va.gov-cms
# VACMS-15842 hide breadcrumb field on all node view displays

## Description

Relates to #15841 & https://dsva.slack.com/archives/CDHBKAL9W/p1698249359564809

## Testing done
Added `breadcrumbs: true` to the `hidden:` section of all node view displays that did not have it. This should have no effect on JSON:API/GraphQL breadcrumb inclusion.

## Screenshots
Prod:
![Screenshot 2023-10-25 at 11 02 09 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/59f233ac-033a-45b7-a598-3d2f8d42a3fb)
![Screenshot 2023-10-25 at 11 02 29 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/32caff79-d033-4690-b718-366ab81f6d9b)

This PR Env:
![Screenshot 2023-10-25 at 11 03 15 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/87adcf2e-7aa8-4a91-9ed1-8d4f3c330e37)
![Screenshot 2023-10-25 at 11 03 23 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/15310ea8-fca3-44b7-bb1d-b5216af0ffef)


## QA steps

- Visit https://pr15858-blwi2kmblbqdrt1glnnkswn0oyhzddkx.ci.cms.va.gov/node/49844/edit and notice no broken link error.
- Visit the view tab for that node and notice no extra breadcrumbs appearing in any of the sections.

- Visit https://pr15858-blwi2kmblbqdrt1glnnkswn0oyhzddkx.ci.cms.va.gov/jsonapi/node/news_story and see that breadcrumbs are still included in the JSON:API response


### Definition of Done


### Select Team for PR review
